andela/ah-frontend-tesseract
# #160814332 Enable user follow/unfollow other users

#### What does this PR do?
Enables a logged in user to see and follow/unfollow other users
#### Description of Task to be completed?
- get list of user
- allow user follow others
- allow user unfollow user he is followng
- give total of followers/following
- write tests
#### How should this be manually tested?
- run `npm start`
- login and navigate to the side bar
- click on users to see all users
#### What are the relevant pivotal tracker stories?
[#160814332](https://www.pivotaltracker.com/story/show/160814332)
#### Screenshots (if appropriate)
<img width="1440" alt="screen shot 2018-10-25 at 16 36 26" src="https://user-images.githubusercontent.com/20810749/47504104-3f068d00-d874-11e8-8dd4-6590eccbb2bd.png">
<img width="1440" alt="screen shot 2018-10-25 at 16 36 19" src="https://user-images.githubusercontent.com/20810749/47504106-3f9f2380-d874-11e8-86ec-416d92774d6e.png">
